src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -889,8 +889,8 @@ javadocTag: AUTHOR_LITERAL (WS | NEWLINE)* ((WS | NEWLINE) description)?
 
       | RETURN_LITERAL (WS | NEWLINE)* ((WS | NEWLINE) description)?
 
-      | SEE_LITERAL (WS | NEWLINE)* reference? (STRING | htmlElement)* (WS | NEWLINE)*
-          ((WS | NEWLINE) description)?
+      | SEE_LITERAL (WS | NEWLINE | {!isNextJavadocTag()}? LEADING_ASTERISK)* reference?
+          (STRING | htmlElement)* (WS | NEWLINE)* ((WS | NEWLINE) description)?
```

Comment:
Description also includes `htmlElement` so is the `htmlElement` needed?

Reply:
it is due to `@see <a href="URL#value">label</a>` see description at http://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#see
but agree that grammar give a bit more freedom than simple `a` tag.

@ps-sp , if javadoc complains about non expected tag as `see` there , please create separate issue on this to update our grammar to allow only `a`.

Reply:
An issue already has been created for at #5036 . Shall be worked on along with #4942 or after it whichever is suitable.